DigitalLearningSolutions.Data.Tests/DataServices/UserDataServiceTests/AdminUserDataServiceTests.cs

```diff
@@ -193,5 +193,29 @@ public void UpdateAdminUserFailedLoginCount_updates_user()
                 transaction.Dispose();
             }
         }
+
+        [Test]
+        public void DeactivateAdminUser_updates_user()
+        {
+            using var transaction = new TransactionScope();
+            var  adminUser = userDataService.GetAdminUserById(7)!;
+            const bool mustBeTrueBeforeDeactivate = true;
+            const bool mustBeFalseAfterDeactivate = false;

```

Comment:
I think these consts are a bit overkill.

Reply:
I removed the constants :)